src/main/java/Entity/Message.java

```diff
@@ -7,6 +7,33 @@ public class Message {
     private String startTime;
     private String endTime;
 
+    //todo
+    public Message(String message) {
```

Comment:
Would the name of the method as a verb give a better idea on what the method does?